base/linalg/dense.jl

```diff
@@ -53,7 +53,7 @@ julia> A
 """
 isposdef!(A::StridedMatrix) = ishermitian(A) && isposdef!(A, :U)
 
-isposdef{T}(A::AbstractMatrix{T}, UL::Symbol) = (S = typeof(sqrt(one(T))); isposdef!(S == T ? copy(A) : convert(AbstractMatrix{S}, A), UL))
+isposdef(A::AbstractMatrix{T}, UL::Symbol) where {T} = (S = typeof(sqrt(one(T))); isposdef!(S == T ? copy(A) : convert(AbstractMatrix{S}, A), UL))
```

Comment:
I meant using the `function` form, since they're multiple statements here introducing intermediate variables